cypress/support/actions/organizations/GroupsAction.ts

```diff
@@ -0,0 +1,66 @@
+import { testData } from 'cypress/fixtures/variables';
+import GroupsRepository from 'cypress/support/repositories/organizations/GroupsRepository';
+
+const groupRepo = new GroupsRepository();
+
+export default class GroupAction {
+  doAddGroup(returnEarly?: boolean) {
+    groupRepo.getAddGroupBtn('addNewGroup').click();
+    groupRepo.getGroupNameInput().type(testData.organizations.groups.newGroupName);
+    groupRepo.getAddGroupSubmitBtn().click();
+
+    cy.wait(3000);
+
+    cy.get('.tableRow').first().should('contain', testData.organizations.groups.newGroupName);
+
+    if (returnEarly) return;
```

Comment:
I wonder about this "returnEarly" variable. It feels like it's misnamed, at the very least.
What it's doing is controlling how many groups are added - I wonder if there's a cleaner way of doing this?

Reply:
refactored it so that it would create two groups, one is used to add user(s) to it, the other one is just used for searching